libs/utils/src/main/java/com/akto/notifications/slack/SlackSender.java

```diff
@@ -0,0 +1,73 @@
+package com.akto.notifications.slack;
+
+import com.akto.dao.context.Context;
+import com.akto.dao.notifications.SlackWebhooksDao;
+import com.akto.dto.notifications.SlackWebhook;
+import com.akto.log.LoggerMaker;
+import com.mongodb.BasicDBObject;
+import com.slack.api.Slack;
+import com.slack.api.webhook.WebhookResponse;
+
+import java.util.List;
+
+public class SlackSender {
+    private static final LoggerMaker loggerMaker = new LoggerMaker(SlackAlerts.class, LoggerMaker.LogDb.DASHBOARD);
+
+    public static void sendAlert(int accountId, SlackAlerts alert) {
+        Thread thread = new Thread(() -> {
```

Comment:
I realised we should use `Executors.newFixedThreadPool`, creating raw threads is expensive in java world